pkg/apiserver/rest/webservice/config.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2022 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webservice
+
+import (
+	restfulspec "github.com/emicklei/go-restful-openapi/v2"
+	"github.com/emicklei/go-restful/v3"
+
+	"github.com/oam-dev/kubevela/apis/types"
+	"github.com/oam-dev/kubevela/pkg/apiserver/log"
+	"github.com/oam-dev/kubevela/pkg/apiserver/model"
+	apis "github.com/oam-dev/kubevela/pkg/apiserver/rest/apis/v1"
+	"github.com/oam-dev/kubevela/pkg/apiserver/rest/usecase"
+	"github.com/oam-dev/kubevela/pkg/apiserver/rest/utils/bcode"
+)
+
+// ConfigWebService returns config web service
+func ConfigWebService(u usecase.ConfigHandler) WebService {
+	return &configWebService{
+		handler: u,
+	}
+}
+
+type configWebService struct {
+	handler usecase.ConfigHandler
+}
+
+func (s *configWebService) GetWebService() *restful.WebService {
```

Comment:
Please add the `authCheckFilter` and `CheckPerm` filters.

Reply:
What are they?

Reply:
Ref:
![image](https://user-images.githubusercontent.com/18493394/160604026-ad3ac8be-c77b-4e86-9329-fa182190977d.png)

Reply:
For resource name definition:

in usecase/rbac.go:L210 
```
"config_types": {
		pathName: "configType",
		subResources: map[string]resourceMetadata{
			"config": {pathName: "configName"},
		},
	},
```